UPGRADE.md

```diff
@@ -0,0 +1,67 @@
+# 3.x Upgrade Guide
+
+Version 3.x features a redesigned Month Picker menu which:
+
+1. Looks more consistent with [jQuery UI Datepicker](https://jqueryui.com/datepicker/).
+2. Turns the Year title into a button which gives the user a "hint" about
+the Jump Years menu when they mouse over it.
+3. Allows the user to return to the year they clicked Jump years at.
+
+See it for yourself at the [demo on JSFiddle](http://jsfiddle.net/kidsysco/JeZap/).
+
```

Comment:
This gives me something to push to kickoff Travic-CI!

good thinking Benjamin!

:D

On Wed, Jan 20, 2016 at 12:59 AM, Benjamin Albert <notifications@github.com>
wrote:

> In UPGRADE.md
> <https://github.com/KidSysco/jquery-ui-month-picker/pull/38#discussion_r50220346>
> :
>
> > @@ -0,0 +1,67 @@
> > +# 3.x Upgrade Guide
> > +
> > +Version 3.x features a redesigned Month Picker menu which:
> > +
> > +1. Looks more consistent with [jQuery UI Datepicker](https://jqueryui.com/datepicker/).
> > +2. Turns the Year title into a button which gives the user a "hint" about
> > +the Jump Years menu when they mouse over it.
> > +3. Allows the user to return to the year they clicked Jump years at.
> > +
> > +See it for yourself at the [demo on JSFiddle](http://jsfiddle.net/kidsysco/JeZap/).
> > +
>
> @KidSysco <https://github.com/KidSysco> Perhaps we should say something
> here about the update being simple, and in some cases will only require
> downloading the new minified version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KidSysco/jquery-ui-month-picker/pull/38/files#r50220346>
> .
>